### ISSUE TITLE
Move repo to ArgoCanada org

### ISSUE THREAD

**Issue by richardsc** [ArgoCanada/argoFloats#73]
Related to #63, I think there are some "admin" things that can only be done by the repo owner (currently @dankelley). It might make sense now that we move the repo over to the new organization [Argo Canada](https://github.com/argoCanada), which will give multiple people "owner" rights on the repo.

@dankelley - can you look into how to do this? I think you go to the "Settings" tab, and there is an option at the bottom to "Transfer ownership". ( I think to use the gh-pages we also need to turn it on on the same page)

**Comment by dankelley** [comment]
Thanks. I'll look at this later today. We are going to make a video in 15 minutes, and I don't want stuff to be broken :-)

**Comment by dankelley** [comment]
Done.  I suppose all three of us should be cloning https://github.com/ArgoCanada/argoFloats now, to avoid confusion.  (I assume we can all still pull and push .. can @j-harbin and @richardsc try please?)

**Comment by dankelley** [comment]
Dear reporter,

Do you think this issue (as defined by its title) has been addressed? If so, please close it. If not, please add a comment explaining what remains to be done. *Thanks!*

PS.  This is a standardized reply.

**Comment by richardsc** [comment]
You can still pull/push as before, though I think that Github will warn you that you should update your source.

To do that, use this command:
```
git remote set-url origin git@github.com:ArgoCanada/argoFloats.git
```

**Comment by richardsc** [comment]
You can verify that it worked by doing:
```
git remote --v
```
```
$ git remote --v
origin  git@github.com:ArgoCanada/argoFloats.git (fetch)
origin  git@github.com:ArgoCanada/argoFloats.git (push)
```

**Comment by richardsc** [comment]
Re-opening because I think that @dankelley is going to need to get the TravisCI and Appveyor thingy-ma-doodles updated?

**Comment by dankelley** [comment]
Good point!  I edited README.  I don't think the travisCI and Appveyor files need to be updated, but there is some magic that has to be done on websites.  I'll try that now, but if I get mixed up, I'll leave that part until tomorrow. (The web stuff changes over time, so I need to struggle to figure it out.)

**Comment by dankelley** [comment]
Yay!  I got travisCI working.  (Don't ask me how.  I'm pretty I did the same thing multiple times but it got working eventually.)

Tomorrow, I'll tackle appveyor and codecov.

**Comment by dankelley** [comment]
Appveyor and codecov are now working (the former in fits and starts ... from some messages, I infer this was a problem of them upgrading R to version 4.0.0, but not updating all packages).

@richardsc --  think this can be reclosed now.

**Comment by dankelley** [comment]
appveyor does not seem to notice "git push" events.  I've fiddled with lots of things, but without any success.  I think the problem arose with the change of repo ownership. TravisCI seemed to handle that smoothly, but appveyor has a different model for identifying projects. 

Frankly, though, the lack of auto-built appveyor does not worry me, since the travisCI test is likely to signal any problems.  Also, I can always trigger an appveyor build with a mouse click, any time I want to (say, maybe, before merging "develop" into "master").

Since I see no path to getting appveyor to work -- trust me, I've spent a lot of time on this -- I feel justified in reclosing this issue.
